python/ray/util/sgd/torch/examples/segmentation/example.yaml

```diff
@@ -0,0 +1,62 @@
+# An unique identifier for the head node and workers of this cluster.
+cluster_name: sgd-coco-pytorch
+
+# The maximum number of workers nodes to launch in addition to the head
+# node. This takes precedence over min_workers. min_workers default to 0.
+min_workers: 0
+initial_workers: 0
+max_workers: 0
+
+target_utilization_fraction: 0.9
+# Cloud-provider specific configuration.
+provider:
+    type: aws
+    region: us-east-1
+    availability_zone: us-east-1c
+
+# How Ray will authenticate with newly launched nodes.
+auth:
+    ssh_user: ubuntu
+
+
+head_node:
+    InstanceType: p3.8xlarge
+    ImageId: ami-0698bcaf8bd9ef56d
+    InstanceMarketOptions:
+        MarketType: spot
```

Comment:
This cannot be preemtible. So we should either specify the block duration (to reserve for 6 hours):
```
SpotOptions:
    BlockDurationMinutes: 360
```
or request an on-demand instance.

Reply:
done